lib/src/screen/dice/dice_screen.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class DiceScreen extends StatefulWidget {
+  DiceScreen({Key key, this.title})
```

Comment:
Remember the trailling commas on the last parameter so Dart formatter helps you with organizing the code

Reply:
Done!